SavingsAccount.java

```diff
@@ -29,15 +30,23 @@ else if (choice == 2) {
             else if (choice == 3) 
             {
                 
+                System.out.print("Withdraw amount:");
+                int withdraw = scanner.nextInt();
+                balance -= withdraw;
+                System.out.println("You have withdrawn:"+ withdraw);
+                System.out.println("Your current balance is:"+ balance);

```

Comment:
Remove this line. There was no requirement that mentioned post withdraw, we need show balance. [YAGNI](https://www.google.com/search?q=yagni&sourceid=chrome&ie=UTF-8)

Reply:
Rather than checking the balance again it displays the updated balance that's the reason I added it.